components/images.css

```diff
@@ -0,0 +1,22 @@
+.img {
+  border-radius: 5px;
+  object-fit: contain;
+}
+.img-thumbnail {
+  max-width: 12rem;
+  max-height: 12rem;
+  border-radius: 4px;
+}
+.img-round {
+  width: 25rem;
+  border-radius: 50%;
```

Comment:
for image round you can use aspect-ration:1/1 property.
You can also use object fit property to fit image properly in parent div.